tests/transfer.test.ts

```
@@ -0,0 +1,170 @@
+  },
+}));
+
+jest.mock("../src/services/stellar/client", () => ({
+  stellarClient: {
+    getServer: jest.fn(),
+    getNetworkPassphrase: jest.fn(() => "Test SDF Network ; September 2015"),
+  },
+}));
+
+jest.mock("../src/config/logger", () => ({
+  logger: { debug: jest.fn(), info: jest.fn(), warn: jest.fn(), error: jest.fn() },
```

ESLint

[error] Replace `·debug:·jest.fn(),·info:·jest.fn(),·warn:·jest.fn(),·error:·jest.fn()` with `⏎····debug:·jest.fn(),⏎····info:·jest.fn(),⏎····warn:·jest.fn(),⏎····error:·jest.fn(),⏎·`

(prettier/prettier)